docs/content/documentation/systems/Materials/tensor_mechanics/ComputeLinearViscoelasticStress.md

```diff
@@ -1,15 +1,24 @@
-# ComputeLinearViscoelasticStress
-This computes the stress of a linear viscoelastic material using a total small strain approximation. The stress is calculated from the mechanical strain and the creep strain as:
+# Compute Linear Viscoelastic Stress
+!syntax description /Materials/ComputeLinearViscoelasticStress
 
-$$
+## Description
+This computes the stress of a linear viscoelastic material using a total small strain approximation. The stress is calculated from the mechanical strain and the creep strain as:
+\begin{equation}
 \sigma_{ij} = C_{ijkl} \left( \epsilon^{mech}_{kl} - \epsilon^{creep}_{kl} \right)
-$$.
+\end{equation}
 
-The mechanical strain must be computed with a [ComputeSmallStrain](/ComputeSmallStrain.md) material. The creep strain itself is computed by a linear viscoelastic material such as a [GeneralizedKelvinVoigtModel](/GeneralizedKelvinVoigtModel.md) or a [GeneralizedMaxwellModel](/GeneralizedMaxwellModel.md) material. The elasticity tensor $C_{ijkl}$ is also provided by the same linear viscoelastic material.
+## Example Input File Syntax
```

Comment:
Should this go near the end of the page?

Reply:
I moved this heading up a bit so that I could include the information about the additional required classes within in the input file example syntax section.